python/pyspark/sql/connect/plan.py

```diff
@@ -322,11 +322,15 @@ def _convert_measure(
     ) -> proto.Aggregate.AggregateFunction:
         exp, fun = m
         measure = proto.Aggregate.AggregateFunction()
-        measure.function.name = fun
+        measure.function.name = fun  # type: ignore[attr-defined]
```

Comment:
Why do these show up as required now, even though we had mypy checks activated before>